pkgs/development/python-modules/gpxpy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, buildPythonApplication, python, lxml }:
+
+buildPythonApplication rec {
```

Comment:
oops, didn't see this earlier, this should be buildPythonPackage if it's in python-modules
```suggestion
buildPythonPackage rec {
```

buildPythonApplication is meant for exporting executables such as ansible

Reply:
Fixed.